pyproject.toml

```diff
@@ -26,8 +26,8 @@ source = ['scanpy']
 omit = ['*/tests/*']
 
 [tool.black]
-line-length = 88
-target-version = ['py36']
+line-length = 120
```

Comment:
This shouldn't change, it's definitely out of scope for the pr, and I think 120 is too long.

It also makes reviewing the diffs in this PR difficult. Could you change this and I'll come back for a review?

Reply:
Then I guess this will a follow up discussion :)

88 makes for some really ugly code imo and 120 is not a lot. But will see what everyone says :)

Reply:
Haha, last time we went through this I was the one pushing hard for increasing it from 80 😆